app/index.html

```diff
@@ -68,18 +68,20 @@
   <script src="_components/FileSaver.min.js"></script>
   <script src="_components/cbio-util.js"></script>
   <script src="_components/download-util.js"></script>
-  <script src="_components/Notification.js"></script>
-  <script src="_components/GeneValidator.js"></script>
-  <script src="_components/QueryByGeneUtil.js"></script>
-  <script src="_components/QueryByGeneTextArea.js"></script>
   <script src="_components/MutatedGeneCNATable.js"></script>
+  <script src="scripts/model/dataProxy.js"></script>
+  <script src="_components/cbioportal-client.js"></script>
   <!-- endbuild -->
 
   <!-- build:js scripts/vc-session.js -->
   <script src="_components/vc-session.js"></script>
   <!-- endbuild -->
 
   <!-- build:js scripts/iviz.js -->
+  <script src="_components/Notification.js"></script>
+  <script src="_components/GeneValidator.js"></script>
+  <script src="_components/QueryByGeneUtil.js"></script>
+  <script src="_components/QueryByGeneTextArea.js"></script>
```

Comment:
I think these files are no longer available in cBioPortal. Either we put them into lib folder, otherwise we need to include in our iViz